app/models/bus_stop.rb

```diff
@@ -50,6 +60,23 @@ def last_updated_by
     User.find_by(id: last_updated.try(:whodunnit)).try :name || 'Unknown'
   end
 
+  def route_list
+    routes.pluck(:number).sort.join(', ')
+  end
+
+  def self.to_csv(limited_attributes: false)
+    attrs = if limited_attributes
+              LIMITED_ATTRS
+            else Hash[columns.map{|c| [c.name, c.name.humanize] }]
```

Comment:
Should I use `human_attribute_name` here to respect locales in the future?

Reply:
I wouldn't worry about it.